<launch>
  <group ns = "carlton">
    <node name="driver" pkg="ardrone_autonomy" type="ardrone_driver" args="-ip 192.168.10.73">	
   </node>
    <node name="teleop" pkg="drone_teleop" type="drone_teleop.py" output="screen" />
    <node name="monitor" pkg="aapacs_drones" type="drone_monitor.py">
      <remap from="image" to="/carlton/ardrone/image_raw" />
    </node>
  </group>
</launch>